egs2/chime4/asr1/conf/train_transformer.yaml

```diff
@@ -0,0 +1,50 @@
+# network architecture
+# encoder related
+encoder: transformer
```

Comment:
Please rename to train_asr_transformer.yaml and train_asr_rnn.yaml

Reply:
Done